meter/lgess.go

```diff
@@ -82,7 +82,7 @@ func NewLgEss(uri, usage, registration, password string, cache time.Duration, ca
 	var setBatteryMode func(api.BatteryMode) error
 	if usage == "battery" {
 		batterySoc = m.batterySoc
-		if version, err := conn.GetFirmwareVersion(); err == nil && version >= 7430 {
+		if version, err := conn.GetFirmwareVersion(); essType != lgpcs.LgEss15 && err == nil && version >= 7430 {
```

Comment:
```suggestion
		if version, err := conn.GetFirmwareVersion(); err == nil && version >= 7430 {
```

Können wir das nicht so lassen? Mit Ess15 hat sich ja noch niemand beschwert?